src/containers/Column/Actions/Actions.js

```diff
@@ -0,0 +1,38 @@
+import React, { PropTypes } from 'react'
+import { Button, Dropdown, Icon } from 'semantic-ui-react'
+
+class Actions extends React.Component {
+  render () {
+    const trigger = (
+      <Button basic>
+        <Icon name='ellipsis horizontal' />
+      </Button>
+    )
+    return (
+      <Button.Group floated='right'>
+        <Dropdown
+          button
+          floating
+          icon={null}
+          labeled
+          options={this.props.actions}
+          trigger={trigger}
+        />
+      </Button.Group>
+    )
```

Comment:
@amandahogan Could definitely just be a button. I went with a dropdown to mimic Trello. Plus I was imagining a bunch of other useful Trello actions we could eventually list in the same menu - Follow All, Unfollow All, Archive This List etc.